robosat/osm/road.py

```diff
@@ -0,0 +1,179 @@
+import sys
+
+import math
+import osmium
+import geojson
+import shapely.geometry
+
+from robosat.osm.core import is_polygon
+
+
+class RoadHandler(osmium.SimpleHandler):
+    """Extracts road polygon features (visible in satellite imagery) from the map.
+    """
+
+    #  highway=* to discard because these features are not vislible in satellite imagery
```

Comment:
This comment is no longer accurate now that we construct the road filter based on thehighway attribute dict. Maybe remove it.